sesame2spiner/examples/aluminum.dat

```diff
@@ -0,0 +1,21 @@
+# ======================================================================
+#  Example input deck for sesame2spiner,
+#  a tool for converting eospac to spiner
+#  Author: Jonah Miller (jonahm@lanl.gov)
+#  © 2021-2023. Triad National Security, LLC. All rights reserved.  This
+#  program was produced under U.S. Government contract 89233218CNA000001
+#  for Los Alamos National Laboratory (LANL), which is operated by Triad
+#  National Security, LLC for the U.S.  Department of Energy/National
+#  Nuclear Security Administration. All rights in the program are
+#  reserved by Triad National Security, LLC, and the U.S. Department of
+#  Energy/National Nuclear Security Administration. The Government is
+#  granted for itself and others acting on its behalf a nonexclusive,
+#  paid-up, irrevocable worldwide license in this material to reproduce,
+#  prepare derivative works, distribute copies to the public, perform
+#  publicly and display publicly, and to permit others to do so.
+# ======================================================================
+
+matid=3720
+name=aluminum
+rhomin = 1e-5
+Tmin = 1
```

Comment:
Mainly it's here because I tested aluminum. Also I found that the minimum density/temperatures provided by the database are frequently unnecessarily low.